src/PHPSemVerCheckerGit/Console/Command/SuggestCommand.php

```diff
@@ -21,222 +21,222 @@
-		}
-
-		$duration = microtime(true) - $startTime;
-		$output->write(
-			[
-				'',
-				'[Scanned files] Before: ' . $before->getFilteredCount() . ' (' . $before->getUnfilteredCount() . ' unfiltered), After: ' . $after->getFilteredCount() . ' (' . $after->getUnfilteredCount() . '  unfiltered)',
-				'Time: ' . round($duration, 3) . ' seconds, Memory: ' . round(memory_get_peak_usage() / 1024 / 1024, 3) . ' MB',
-			],
-			true
-		);
-		return 0;
-	}
-
-	/**
-	 * @param \PHPSemVerChecker\Report\Report $report
-	 * @param \vierbergenlars\SemVer\version  $tag
-	 * @return \vierbergenlars\SemVer\version
-	 */
-	private function getNextTag(Report $report, SemanticVersion $tag)
-	{
-		$newTag = new SemanticVersion($tag);
-		$suggestedLevel = $report->getSuggestedLevel();
-		if ($suggestedLevel === Level::NONE) {
-			return $newTag;
-		}
-		if ($newTag->getPrerelease()) {
-			$newTag->inc('prerelease');
-			return $newTag;
-		}
-		if ($newTag->getMajor() < 1 && $suggestedLevel === Level::MAJOR) {
-			$newTag->inc('minor');
-			return $newTag;
-		}
-		$newTag->inc(strtolower(Level::toString($suggestedLevel)));
-		return $newTag;
-	}
-
-	/**
-	 * @param \Gitter\Repository $repository
-	 * @return null|string
-	 */
-	private function getInitialTag(Repository $repository)
-	{
-		$tag = $this->config->get('tag');
-		if ($tag === null) {
-			return $this->findLatestTag($repository);
-		}
-		return $this->findTag($repository, $tag);
-	}
-
-	/**
-	 * @param \Gitter\Repository $repository
-	 * @return string|null
-	 */
-	protected function findLatestTag(Repository $repository)
-	{
-		return $this->findTag($repository, '*');
-	}
-
-	/**
-	 * @param \Gitter\Repository $repository
-	 * @param string             $tag
-	 * @return string|null
-	 */
-	protected function findTag(Repository $repository, $tag)
-	{
-		$tags = (array)$repository->getTags();
-		$tags = $this->filterTags($tags);
-
-		$tagExpression = new SemanticExpression($tag);
-
-		try {
-			// Throws an exception if it cannot find a matching version
-			$satisfyingTag = $tagExpression->maxSatisfying($tags);
-		} catch (SemanticVersionException $e) {
-			return null;
-		}
-
-		return $this->getMappedVersionTag($tags, $satisfyingTag);
-	}
-
-	/**
-	 * @param array $tags
-	 * @return array
-	 */
-	private function filterTags(array $tags)
-	{
-		$filteredTags = [];
-		foreach ($tags as $tag) {
-			try {
-				new SemanticVersion($tag);
-				$filteredTags[] = $tag;
-			} catch (SemanticVersionException $e) {
-				// Do nothing
-			}
-		}
-		return $filteredTags;
-	}
-
-	/**
-	 * @param string[]                                   $tags
-	 * @param \vierbergenlars\SemVer\version|string|null $versionTag
-	 * @return string|null
-	 */
-	private function getMappedVersionTag(array $tags, $versionTag)
-	{
-		foreach ($tags as $tag) {
-			try {
-				if (SemanticVersion::eq($versionTag, $tag)) {
-					return $tag;
-				}
-			} catch (RuntimeException $e) {
-				// Do nothing
-			}
-		}
-		return null;
-	}
+    /**
```

Comment:
**MUST**
Let's keep the tab indentation. Migration to PSR-2 will be done in a separate PR.

Reply:
Resolved now. This file is untouched.